lab-meetings/README.md

```diff
@@ -11,6 +11,28 @@ The lab also has a [google calendar](https://calendar.google.com/calendar/embed?
 The hackmd links for each meeting are not publicly available as they contain some conversations that lab members would prefer to keep private.
 As a reminder, please **complete the HackMD in advance of the meeting** in order to facilitate conversations before and during the 1 hour time slot.
 
+### Summer 2020
+
+This table covers lab meetings from June 2020 to August 2020.
+
+| Date | Time | Chair | Topic | Link to blog |
+| ---- | ----- | ----- | ----- | ------------ |
+| 4 June | - | No lab meeting | [Challenging power in data science](https://www.turing.ac.uk/events/challenging-power-data-science) talk |  |
+| 11 June | 9:00 | Kirstie | Open agenda | |
+| 18 June | - | No lab meeting | |
+| 25 June | 9:00 | Ismael | Ethics Advisory Group training plan | |
+| 2 July | 15:00 | Louise | Reproducibility | | 
+| 9 July | 9:00 | Sophia | Introduction and general musings! | | 
+| 16 July | 15:00 | - |  No lab meeting | [Data Science for Mental Health Interest Group](https://turing-ds4mh.github.io/meetings.html), Guardian online [Book Club with Jeanette Winterson](https://membership.theguardian.com/event/book-club-with-jeanette-winterson-108864727330) at 7pm | |
+| 23 July | 9:00 | Sarah & Yo | Continuous Integration and GitHub Actions | |
```

Comment:
This clashes with the first day of EuroPython for me and 30th July is JuliaCon. I haven't seen full schedules so it might be that I just miss an hour, but maybe also push back to somewhere between the summer breaks? @yochannah?

Reply:
It's also kinda tricky for me - that is my last day at InterMine :) 30th I'll be on vacation between jobs so it would be easier (and after that I'll be at new job, so it's hard to commit until I know more about my schedule).

Reply:
Ah! Sure - we can push to August or swap with @brainonsilicone... what do you think @yochannah and @sgibson91 ?

Reply:
My preference is August as I still have to prepare for EuroPython and JuliaCon.

Reply:
okay - let's swap 'til august but beware that I don't know my schedule yet :) It might be fine.

Reply:
I'm going to keep you out of the schedule for now.... I feel like august is going to get a bit rogue. We can always do a long coffee hour once you're prepped @sgibson91 !